cypress/integration/dib-travel/company-settings/reference-fields/helpers/delete-cost-center.ts

```diff
@@ -0,0 +1,14 @@
+export const deleteCostCenter = (referenceFields: string) => {
+  cy.waitForAngular();
```

Comment:
Can you confirm if we still need this waitForAngular? Maybe this can be removed (please confirm smaller timeout)